net/bridge.go

```diff
@@ -328,21 +329,26 @@ func (b bridgeImpl) initPrep(config *BridgeConfig) error {
 	// instead we create a temporary interface with the desired MTU,
 	// attach that to the bridge, and then remove it again.
 	dummy := &netlink.Dummy{LinkAttrs: netlink.NewLinkAttrs()}
-	dummy.LinkAttrs.Name = "vethwedu"
-	if err = netlink.LinkAdd(dummy); err != nil {
+	dummy.LinkAttrs.Name = WeaveDummyIfName
+	if err = LinkAddIfNotExist(dummy); err != nil {
 		return errors.Wrap(err, "creating dummy interface")
 	}
+	defer func() {
+		var dummy netlink.Link
+		dummy, err = netlink.LinkByName(WeaveDummyIfName)
```

Comment:
can we just re-use the `dummy` variable that was initialized earlier, instead of creating a new one?

Reply:
I think you don't even have to assign to it - the initialization earlier leaves `dummy` in a state where you can just use it.  Unless I'm missing something?

Reply:
Right. I changed the logic to re-use `dummy`. PTAL